modules/plugin/jdbc/jdbc-oracle/src/main/java/org/geotools/data/oracle/OracleNGDataStoreFactory.java

```diff
@@ -124,6 +124,15 @@ public class OracleNGDataStoreFactory extends JDBCDataStoreFactory {
                     "Specifies the timeout when negotiating a session with the database listener (milliseconds)",
                     false);
 
+    /** Specifies whether column REMARKS metadata will be returned. */
+    public static final Param GET_COLUMN_REMARKS =
+            new Param(
+                    "Get column remarks",
+                    Boolean.class,
+                    "Specifies whether column REMARKS metadata will be returned",
+                    false,
+                    Boolean.TRUE);
```

Comment:
Out of caution, I'd keep the default to FALSE getting the remarks should be costing some if Oracle defaults not to do that (and might also be related to permissions in some way).

```suggestion
                    Boolean.FALSE);
```